conda.recipe/meta.yaml

```diff
@@ -19,6 +19,8 @@ requirements:
     - python {{ python }}
     - setuptools
     - setuptools_scm
+    - eigen
+    - xsimd
```

Comment:
I think we need to add `pybind11` here, and remove it from the other place where it is being installed.